src/WinGetYamlFuzzing/README.md

```diff
@@ -1,11 +1,30 @@
 # WinGetYamlFuzzing
-The goal of this project is to create a [libFuzzer](http://llvm.org/docs/LibFuzzer.html) based fuzzer for our YAML manifest loading.
+The goal of this project is to create a [libFuzzer](http://llvm.org/docs/LibFuzzer.html) based fuzzer for our YAML manifest parsing.
 
 ## Issues
-Currently the fuzzer crashes when exceptions are thrown (built using the VS clang 10 package). This is suspected to be caused by the issue mentioned [here](https://github.com/google/oss-fuzz/issues/2328),
-which while fixed, was also regressed. While investigation continues, the fuzzer is of little value.
+There is a known issue that is fixed in LLVM 12 for running on Windows, and the only known method to build a successful fuzzer is to manually build a local copy of libFuzzer and link it in to this project.
+
+## Building
+
+First, clone https://github.com/llvm/llvm-project/tree/llvmorg-12.0.0-rc4 (last known working).
+
+From the local clone, run these commands in `cmd` (modifying VS install location as needed):
+```
+mkdir build
+cd build
+set verbose=1
+call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+cmake -DLLVM_ENABLE_PROJECTS="clang;compiler-rt" -DCOMPILER_RT_BUILD_BUILTINS=OFF -DCOMPILER_RT_BUILD_CRT=OFF -DCOMPILER_RT_BUILD_SANITIZERS=OFF -DCOMPILER_RT_BUILD_XRAY=OFF -DCOMPILER_RT_BUILD_PROFILE=OFF -DLLVM_TARGETS_TO_BUILD=X86 -Thost=x64 ../llvm

```

Comment:
If you're feeling adventurous, you might be able to add this to `patterns.txt`:

```
# compile flags
 -[TD]
```
(The leading space is important)